daprdocs/content/en/developing-applications/building-blocks/jobs/howto-handle-triggered-jobs.md

```diff
@@ -0,0 +1,223 @@
+---
+type: docs
+title: "How-To: Handle triggered jobs"
```

Comment:
- Better to show the Go SDK methods rather than HTTP
- the code here should be a snippet that simply shows the main methods calls that you want to person to focus on. There are already full code samples here https://github.com/dapr/docs/pull/4276 otherwise we are just repeating everything. 
- This HowTo (which tells you the essential points) then simply points to this code example
- Have a couple of sentences telling the person what is happening. Also does it really need to be split into two apps? People will think is what is required, when this all can be in the same app and will be 90% of the time. 
- Also do not see why this should be split into 2 different topics. Just have one topic on "How to schedule and handle trigger jobs" (include scheduling and triggers)